### PR TITLE
Update books.js

### PR DESCRIPTION
**Pull request by bhskr44**
# Highlights
✔️ Linters Fixed
✔️ Redux Added
✔️Add Remove Functionality Added

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { uuid4 } from 'uuid4';
```
Comment (rustam-tolipov) [inline src/redux/books/books.js:1]: **You did a great job! However, there is one part that needs to be changed to make your code approve.**

- [x] That part of your code is not created as it is meant to be. You need to get values (onChange) by updating localStates with (useState). Kindly change this implementation from DOM to more React way 😉
More info: [highlight](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components.md#:~:text=Set%20the%20data%20inputs%20in%20the%20local%20React%20state%20(set%20title%20and%20author%20%2D%20NOTE%3A%20categories%20and%20comments%20are%20NOT%20part%20of%20this%20step).)

Screenshot 2022-12-16 at 02 24 40

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
```
Comment (rustam-tolipov) [inline src/redux/books/books.js:2]: [OPTIONAL] When I fill in input and create a new book it's working. But, if I try to create a new book again the input field is filled again with the previous words. I suggest you remove them when we submit to create a new book.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @bhskr44,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Bhaskar, 
## To Highlight 🎉
App works as expected. :heavy_check_mark:
Readme file is professional. :heavy_check_mark:
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request closed without merging**